source/common/Release_Specific/_Processor_SDK_Supported_Platforms_and_Versions.rst

```diff
@@ -235,15 +235,30 @@ The following platforms and EVMs are supported with Processor SDK.
 
 .. ifconfig:: CONFIG_part_family in ('AM57X_family')
 
-    +--------------+-----------+-----------+-----------------------+-------------------+------------------+
-    | **Platform** | **EVM**   | **Tested  | **Document**          | **PSDK Linux**    | **PSDK RTOS**    |
-    |              |           | Version** |                       |                   |                  |
-    +--------------+-----------+-----------+-----------------------+-------------------+------------------+
-    | AM57X        | AM572x EVM| A3        | Hardware User's Guide | Y                 | Y                |
-    +--------------+-----------+-----------+-----------------------+-------------------+------------------+
-    | AM57X        | AM572x EVM| 1.3A      | Hardware User's Guide | Y                 | Y                |
-    +--------------+-----------+-----------+-----------------------+-------------------+------------------+
-    | AM57         | AM571x IDK| 1.3A      | Hardware User's Guide | Y                 | Y                |
-    +--------------+-----------+-----------+-----------------------+-------------------+------------------+
-    | AM57         | AM574x IDK| 1.0A      | Hardware User's Guide | Y                 | Y                |
-    +--------------+-----------+-----------+-----------------------+-------------------+------------------+
+    +--------------+-----------+-----------+-------------------+------------------+
+    | **Platform** | **EVM**   | **Tested  | **PSDK Linux**    | **PSDK RTOS**    |
+    |              |           | Version** |                   |                  |
+    +--------------+-----------+-----------+-------------------+------------------+
+    | AM57X        | AM572x EVM| A3        | Y                 | Y                |
+    +--------------+-----------+-----------+-------------------+------------------+
+    | AM57X        | AM572x EVM| 1.3A      | Y                 | Y                |
+    +--------------+-----------+-----------+-------------------+------------------+
+    | AM57         | AM571x IDK| 1.3A      | Y                 | Y                |
+    +--------------+-----------+-----------+-------------------+------------------+
+    | AM57         | AM574x IDK| 1.0A      | Y                 | Y                |
+    +--------------+-----------+-----------+-------------------+------------------+
+
+    Document reference:
+
+    +-----------------------------------------------------+---------------------------------------------------------------+
+    | **EVM**                                             | **Document**                                                  |
+    +-----------------------------------------------------+---------------------------------------------------------------+
+    |`AM572x EVM <https://www.ti.com/tool/TMDSEVM572X>`__ |`Hardware User's Guide <https://www.ti.com/lit/pdf/spruig1>`__ |
```

Comment:
Please make sure that there is at least one space around vertical bars in tables to prevent accidental processing as a replacement. Alternatively, use list-tables